CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 ## [UNRELEASED] - YYYY-MM-DD
 ### Added
 - Add new `case_sensitive` parameter to `match_streaminfos`, defaulting to `False` to maintain previous behavior; when `False`, stream properties are matched more leniently ([#134](https://github.com/xdf-modules/pyxdf/pull/134) by [Stefan Appelhoff](https://github.com/sappelhoff))
+- Expand tests for dejitter segments to cover all initialisation contexts ([#129](https://github.com/xdf-modules/pyxdf/pull/129) by [Jamie Forth](https://github.com/jamieforth))
+- Generic `MockStreamData` class for unit testing ([#130](https://github.com/xdf-modules/pyxdf/pull/130) by [Jamie Forth](https://github.com/jamieforth))
+- New tests for jitter removal and segmentation ([#130](https://github.com/xdf-modules/pyxdf/pull/130) by [Jamie Forth](https://github.com/jamieforth))
```

Comment:
I don't think we need to include additions/changes to our tests here (at least we've never done that before).

Reply:
There was a previous entry about tests so I just followed that, but yes I don't think it's necessary to document tests here.